### ISSUE TITLE
Custom work items apperance 

### ISSUE THREAD

**Issue by Danne-S** [Microsoft/vsts-zendesk-app#20]
Hi,

I have created a custom work item and added it to the same level as PBI and BUGs. Now I want to be able to create a work item of this type from Zendesk. I hoped it would work immediately but it seems like some changes are needed.

**Comment by bfcamara** [comment]
You're totally right - currently the app only supports a fixed list of work item types: Bug, Product Backlog Item, User Story, Requirement, and Issue. 

I can imagine a new configuration setting where the user could specify extra custom work item types allowed. It would work but the custom work item types couldn't have new required fields, otherwise the work item creation would fail.
